201-decrypt-vmss-linux/prereqs/prereq.azuredeploy.json

```diff
@@ -13,6 +13,13 @@
             "metadata": {
                 "description": "Number of desired VM instances"
             }
+        },
```

Comment:
format the JSON (e.g. SHIFT+ALT+F in VS Code)

Reply:
Fixed in 2e2c891